acceptance_tests/features/steps/sms_fulfilment.py

```diff
@@ -1,19 +1,21 @@
 import uuid
-from typing import List
 
 import requests
 from behave import step
 
-from acceptance_tests.utilities import template_helper
 from acceptance_tests.utilities.audit_trail_helper import get_unique_user_email
 from acceptance_tests.utilities.event_helper import get_emitted_survey_update_by_id
 from acceptance_tests.utilities.notify_helper import check_sms_fulfilment_response
 from acceptance_tests.utilities.test_case_helper import test_helper
 from config import Config
 
 
-@step("fulfilments are authorised on sms template")
-def authorise_sms_pack_code(context):
+@step("fulfilments are authorised for sms template {template_name}")
+def authorise_sms_pack_code(context, template_name):
+    context.template = context.sms_templates[template_name.strip('\"')]['template']
+    context.pack_code = context.sms_packcodes[template_name.strip('\"')]['pack_code']
+    context.notify_template_id = context.sms_packcodes[template_name.strip('\"')]['notify_template_id']
```

Comment:
Rather than stripping the quotes, you can bake them into the step definition like 
```python
@step('fulfilments are authorised for sms template "{template_name}"')
```